src/Raven.Server/Documents/Replication/ReplicationItems/DocumentReplicationItem.cs

```diff
@@ -20,7 +20,7 @@ public sealed class DocumentReplicationItem : ReplicationBatchItem
         public override DynamicJsonValue ToDebugJson()
         {
             var djv = base.ToDebugJson();
-            djv[nameof(Collection)] = Collection.ToString(CultureInfo.InvariantCulture);
+            djv[nameof(Collection)] = Collection?.ToString(CultureInfo.InvariantCulture);
```

Comment:
why is that `null`?

Reply:
I checked and it can be null when we store a document with "empty" collection. for example:

```
using (var session = source.OpenAsyncSession())
{
    await session.StoreAsync(new { Name = "Foo" }, "FoObAr/0");
    await session.SaveChangesAsync();
}
```

Reply:
done